packages/caip/src/adapters/coingecko/utils.test.ts

```diff
@@ -69,6 +70,13 @@ const makePolygonMockCoingeckoResponse = () => ({
   platforms: {},
 })
 
+const makeGnosisMockCoingeckoResponse = () => ({
+  id: 'gnosis',
+  symbol: 'xDai',
+  name: 'Gnosis',
```

Comment:
Still incorrect, it's also the name of the token, though cased differently in the case of Coingecko API, see https://api.coingecko.com/api/v3/coins/xdai

Reply:
good catch, funny thing is we are only checking the tokens for these responses so not actually using this data which makes these mocks pretty pointless. it is validating against the gnosis baseAsset which is why the tests still pass. Could just remove

Reply:
I've just removed the names

Reply:
We could really just remove all of the mocks that don't include any of the `platforms` details

Reply:
Resolving for now so we can merge this, @lmyslinski let's remove the irrelevant mocks as a follow-up